includes/admin/class-gmc-admin.php

```diff
@@ -532,13 +532,14 @@ public function cpt2_metaboxes_fields() {
 			) ),
 		) );
 		$this->display_options->add_field( array(
-			'name' => __( 'Custom Map Theme JSON', 'google-maps-builder' ),
-			'desc' => __( 'Paste the Snazzy Map JSON code into the field above to set the theme.', 'google-maps-builder' ),
-			'id'   => $prefix . 'theme_json',
-			'type' => 'textarea_code',
+			'name'       => __( 'Custom Map Theme JSON', 'google-maps-builder' ),
+			'id'         => $prefix . 'theme_json',
+			'type'       => 'textarea_code',
+			'attributes' => array(
+				'placeholder' => __( 'Paste the custom theme Json/JS code here.', 'google-maps-builder' ),
```

Comment:
Capitalize `JSON`

Reply:
@DevinWalker as per suggestion I have done the changes.

Thanks